src/event.js

```diff
@@ -181,60 +188,25 @@ PubfoodEvent.prototype.publish = function(eventType, data, eventContext) {
   });
 };
 
-/**
- * Emit an event to all registered event listeners.
```

Comment:
no real point having `@private` jsdoc referencing external git repo.

added specific jsdoc to extended EventEmitter functions.